app/controllers/planning_application/review_immunity_details_controller.rb

```diff
@@ -22,35 +23,50 @@ def edit
     end
 
     def update
```

Comment:
Thinking we could add a `before_action` to check that the current user is a reviewer. Definitely don't want an assessor being able to review and approve their own assessment.